src/Data/Entities/AdminEntity.cs

```diff
@@ -0,0 +1,14 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace Data.Entities;
+
+public class AdminEntity {
+    [Key] public Guid Id { get; set; }
+    
+    [Required]
+    [EmailAddress]
+    public string Email { get; set; }
+
+    [Required]
+    public string Password { get; set; }
```

Comment:
Do we want Password as a public

Reply:
Well, its a property. And whatever comes here is hashed already. And it needs to be a public property for EFC to work.